Runtime/UIElements/VisualElementExtension.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Reflection;
+using UnityEngine.UIElements;
+
+namespace StansAssets.Foundation.UIElements
+{
+    /// <summary>
+    /// Public representation of internal VisualElement API.
```

Comment:
I think in the long run we will also have more utilities in this class. So I would say VisualElement Utility Methods.
Just to be consistent
https://api.stansassets.com/foundation/StansAssets.Foundation.EnumUtility.html